pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "dynamic_models"
-version = "0.4.0"
+version = "0.4.1"
```

Comment:
I haven't released 0.4.0 so it's probably safe to keep the version the same so the package doesn't jump from 0.3.0 straight to 0.4.1